### PR TITLE
Add display set to answer serializer

### PR DESCRIPTION
**Pull request by MikeOverkamp-diag**

### CODE REVIEW

File: app/grandchallenge/reader_studies/serializers.py [hunk 1]
```diff
@@ -157,30 +168,43 @@ def validate(self, attrs):
...
         if self.instance:
-            on_commit(
-                lambda: add_scores.apply_async(
-                    kwargs={
-                        "instance_pk": str(self.instance.pk),
-                        "pk_set": list(
-                            map(str, images.values_list("pk", flat=True))
-                        ),
-                    }
+            if images is not None and images.count() > 0:
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/serializers.py:188]: Shouldn't we validate that only `images` or `display_set` are set?

File: app/grandchallenge/reader_studies/tasks.py [hunk 2]
```diff
@@ -46,6 +46,27 @@ def add_scores(*, instance_pk, pk_set):
             add_score(instance, ground_truth.answer)
 
 
+@shared_task
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/tasks.py:49]: Please do not use the default queue. Please ensure that any new tasks are idempotent and use one of the acks-late queues (depending on their memory/duration requirements).

**Review by jmsmkn — COMMENTED** [review]
Looks good! Some minor comments.

File: app/grandchallenge/reader_studies/tasks.py (hunk 2 above)
Comment (MikeOverkamp-diag) [reply app/grandchallenge/reader_studies/tasks.py]: I think the micro-short queue should be sufficient for this

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**